src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -71,6 +71,7 @@ public String weakMessageEncryption(String message, String key) throws Exception
     }
 
     public ShopifyShop connectToShopify(String subdomain) {
+        final String token2 = "shpss_sdkfhkjh134134141341344133412312345678";
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded Shopify Shared Secret. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)